src/Workspaces/CoreTestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -305,7 +305,7 @@ public override int WriteTimeout
                 public override Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken) => _stream.ReadAsync(buffer, offset, count, cancellationToken);
                 public override Task WriteAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken) => _stream.WriteAsync(buffer, offset, count, cancellationToken);
 
-#if NET5_0 // nullability annotations differ
+#if NET5_0_OR_GREATER // nullability annotations differ
```

Comment:
Any reason for this form vs. `#if NETCOREAPP`? Want to make sure there isn't a `netcoreapp3.1` usage still hanging around.